src/Http/Kernel.php

```diff
@@ -87,6 +84,20 @@ public function __construct(Application $app, Router $router)
         parent::__construct($app, $router);
     }
 
+    /**
+     * Handle an incoming HTTP request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function handle($request)
```

Comment:
```php
public function handle(Request $request)
```

Reply:
I've kept the `handle()` method un-hinted as the parent method in `Illuminate\Foundation\Http\Kernel` is also un-hinted.